src/main/java/io/github/eliux/mega/cmd/ExportInfo.java

```diff
@@ -10,22 +16,44 @@
  */
 public class ExportInfo {
 
+    private static String dateFormat = "EEE, dd MMM yyyy HH:mm:ss Z";
     private static final Pattern LIST_PATTERN =
             Pattern.compile("(?<remotePath>\\S+) \\(.+link: (?<publicLink>http[s]?://mega.nz/\\S*#.+)\\)");
 
+    private static final Pattern EXPORT_PATTERN =
+            Pattern.compile("\\s(?<remotePath>\\S+)(:)\\s(?<publicLink>http[s]?://mega.nz/\\S*#\\S*)( expires at (?<expireDate>.{3}, .{2} .{3} .{4} .{2}:.{2}:.{2} .+))?");
+
     private final String remotePath;
 
     private final String publicLink;
 
+    private Optional<String> expireDate;
+
     public ExportInfo(String remotePath, String publicLink) {
         this.remotePath = remotePath;
         this.publicLink = publicLink;
     }
 
     public static ExportInfo parseExportInfo(String exportInfoStr) {
-        final String[] tokens = exportInfoStr.replace("Exported ", "")
-                .split(": ");
-        return new ExportInfo(tokens[0], tokens[1]);
+        final Matcher matcher = EXPORT_PATTERN.matcher(exportInfoStr);
+
+        if (matcher.find()) {
+            String remotePath = matcher.group("remotePath");
+            String publicLink = matcher.group("publicLink");
+            String expireDate = matcher.group("expireDate");
+
+            if (expireDate == null && exportInfoStr.contains("expires at")) {
```

Comment:
I think it's not possible to have "expires at" without a date but it's possible that they change the format. In that case the expireDate will be null but "exportInfoStr" will still contains "expires at".

Reply:
In such case I would not do such strict parsing to the format of the obtained date. What do you think about getting whatever is the text that comes after "expires at" and try to create a date with it and catch an exception if it is not possible?

Reply:
Check out whether `io.github.eliux.mega.MegaUtils#parseFileDate` works for that format otherwise create another function alike there in `MegaUtils`.

Reply:
Seems right, i just checked the method in `MegaUtils`, it's not the same date format but i can definetly create a new format alike.

Reply:
> I think it's not possible to have "expires at" without a date but it's possible that they change the format. In that case the expireDate will be null but "exportInfoStr" will still contains "expires at".

Having into account that after "expires at" there will be a date (with a valid format or not) I think that when you run the find with the regexp `EXPORT_PATTERN` you will get a value if this was specified.
In case that the specified value could not be parsed an automated test should detect this issue by throwing an uncatched exception so you do not need to check `.contains("expires at")`. 

I understand that such expiration date is optional (the regexp also indicates it) but if the regexp could not catch it, why it would be different with you checking it with a `String#contains`? It looks like a code smell, because it seems that you are not sure that your regexp is going to be trustworthy. I just recommend you to erase this part `{3}, .{2} .{3} .{4} .{2}:.{2}:.{2} .+`  and just rely on `dateFormat` to validate whether it is the right date format. Also convert dateFormat to a constant:
```    
private static final String DATE_FORMAT= "EEE, dd MMM yyyy HH:mm:ss Z";
```